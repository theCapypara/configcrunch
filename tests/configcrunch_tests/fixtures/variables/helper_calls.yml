base:
  level_direct:
    name: "{{ parent().more.b }}"
  more:
    a: "{{ simple_helper() }}"
    b: from_base
    c: "{{ parent().more.b }}"