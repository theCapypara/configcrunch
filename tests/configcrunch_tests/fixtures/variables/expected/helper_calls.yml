base:
  level_direct:
    name: from_base
  more:
    a: simple
    b: from_base
    c: from_base